tests/test_ingest_covid19dp_submission.py

```diff
@@ -0,0 +1,143 @@
+        spring.main.allow-bean-definition-overriding=true
+        #As this is a spring batch application, disable the embedded tomcat. This is the new way to do that for spring 2.
+        spring.main.web-application-type=none
+        spring.main.web-environment=false
+
+        spring.jpa.generate-ddl=true
+        # This entry is put just to avoid a warning message in the logs when you start the spring-boot application.
+        # This bug is from hibernate which tries to retrieve some metadata from postgresql db and failed to find that and logs as a warning
+        # It doesnt cause any issue though.
+        spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation = true
+        """
+
+    def setUp(self) -> None:
+        shutil.rmtree(self.download_folder, ignore_errors=True)
+        shutil.rmtree(self.processing_folder, ignore_errors=True)
+        os.makedirs(self.processing_folder)
+
+        run_command_with_output("Downloading accessioning JAR file...",
+                                f'bash -c '
+                                f'"cd {self.processing_folder} && git clone https://github.com/EBIVariation/eva-accession '
+                                f'&& cd eva-accession && mvn -q package -DskipTests '
+                                f'&& cp eva-accession-pipeline/target/*.jar '
+                                f'{self.processing_folder} '
+                                f'&& cp eva-accession-clustering/target/*.jar '
+                                f'{self.processing_folder} '
+                                f'&& cd {self.processing_folder} && rm -rf eva-accession"')
+        self.accession_jar_file = glob.glob(f"{self.processing_folder}/eva-accession-pipeline*.jar")[0]
+        self.clustering_jar_file = glob.glob(f"{self.processing_folder}/eva-accession-clustering*.jar")[0]
+        open(self.accessioning_properties_file, "w").write(self.accessioning_properties)
+        open(self.clustering_properties_file, "w").write(self.clustering_properties)
+        self.app_config = {"app": {"bcftools_binary": "bcftools", "nextflow_binary": "nextflow",
+                                   "validator_binary": "vcf_validator_linux",
+                                   "accessioning_jar_file": self.accession_jar_file,
+                                   "accessioning_properties_file": self.accessioning_properties_file,
+                                   "accessioning_instance": "instance-10",
+                                   "clustering_jar_file": self.clustering_jar_file,
+                                   "clustering_properties_file": self.clustering_properties_file},
+                           "submission": {"concat_chunk_size": 100,
+                                          "ftp_project_dir": f"{self.download_folder}/ftp"}}
+        yaml.safe_dump(data=self.app_config, stream=open(self.app_config_file, "w"))
```

Comment:
I actually kind of like having these config & properties files in the `tests/resources` if possible, it can serve as a nice example for other people and saves having to clean-up the files after the tests.

Reply:
It also would save you from having multiple copies of the accessioning & clustering properties floating around in your tests.